x-pack/plugin/sql/qa/server/src/main/resources/agg.csv-spec

```diff
@@ -1325,3 +1325,386 @@ F              |1964-10-18T00:00:00.000Z|1952-04-19T00:00:00.000Z
+SELECT STDDEV_POP(bytes_in) as "STDDEV_POP_AllZeros" FROM logs WHERE bytes_in = 0;
+
+STDDEV_POP_AllZeros
+-------------------
+0.0                
+;
+
+
+allNullsWithStddevPop
+schema::STDDEV_POP_AllNulls:d
+SELECT STDDEV_POP(bytes_out) as "STDDEV_POP_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+STDDEV_POP_AllNulls
+-------------------
+null               
+;
+
+
+allZerosWithStddevSamp
+schema::STDDEV_SAMP_AllZeros:d
+SELECT STDDEV_SAMP(bytes_in) as "STDDEV_SAMP_AllZeros" FROM logs WHERE bytes_in = 0;
+
+STDDEV_SAMP_AllZeros
+--------------------
+0.0                 
+;
+
+
+allNullsWithStddevSamp
+schema::STDDEV_SAMP_AllNulls:d
+SELECT STDDEV_SAMP(bytes_out) as "STDDEV_SAMP_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+STDDEV_SAMP_AllNulls
+--------------------
+null                
+;
+
+
+allZerosWithVarSamp
+schema::VAR_SAMP_AllZeros:d
+SELECT VAR_SAMP(bytes_in) as "VAR_SAMP_AllZeros" FROM logs WHERE bytes_in = 0;
+
+VAR_SAMP_AllZeros
+-----------------
+0.0              
+;
+
+
+allNullsWithVarSamp
+schema::VAR_SAMP_AllNulls:d
+SELECT VAR_SAMP(bytes_out) as "VAR_SAMP_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+VAR_SAMP_AllNulls
+-----------------
+null             
+;
+
+
+allZerosWithVarPop
+schema::VAR_POP_AllZeros:d
+SELECT VAR_POP(bytes_in) as "VAR_POP_AllZeros" FROM logs WHERE bytes_in = 0;
+
+VAR_POP_AllZeros
+----------------
+0.0             
+;
+
+
+allNullsWithVarPop
+schema::VAR_POP_AllNulls:d
+SELECT VAR_POP(bytes_out) as "VAR_POP_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+VAR_POP_AllNulls
+----------------
+null            
+;
+
+
+allZerosWithSkewness
+schema::SKEWNESS_AllZeros:d
+SELECT SKEWNESS(bytes_in) as "SKEWNESS_AllZeros" FROM logs WHERE bytes_in = 0;
+
+SKEWNESS_AllZeros
+-----------------
+NaN              
+;
+
+
+allNullsWithSkewness
+schema::SKEWNESS_AllNulls:d
+SELECT SKEWNESS(bytes_out) as "SKEWNESS_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+SKEWNESS_AllNulls
+-----------------
+null             
+;
+
+
+allZerosWithMad
+schema::MAD_AllZeros:d
+SELECT MAD(bytes_in) as "MAD_AllZeros" FROM logs WHERE bytes_in = 0;
+
+ MAD_AllZeros  
+---------------
+0.0            
+;
+
+
+allNullsWithMad
+schema::MAD_AllNulls:d
+SELECT MAD(bytes_out) as "MAD_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+ MAD_AllNulls  
+---------------
+NaN            
+;
+
+
+allZerosWithKurtosis
+schema::KURTOSIS_AllZeros:d
+SELECT KURTOSIS(bytes_in) as "KURTOSIS_AllZeros" FROM logs WHERE bytes_in = 0;
+
+KURTOSIS_AllZeros
+-----------------
+NaN              
+;
+
+
+allNullsWithKurtosis
+schema::KURTOSIS_AllNulls:d
+SELECT KURTOSIS(bytes_out) as "KURTOSIS_AllNulls" FROM logs WHERE bytes_out IS NULL;
+
+KURTOSIS_AllNulls
+-----------------
+null             
+;
+
+nullsAndZerosCombined
+schema::COUNT(*):l|COUNT_AllZeros:l|COUNT_AllNulls:l|FIRST_AllZeros:i|FIRST_AllNulls:i|SUM_AllZeros:i|SUM_AllNulls:i
+SELECT
+    COUNT(*)
+    , COUNT(bytes_in) AS "COUNT_AllZeros"
+    , COUNT(bytes_out) AS "COUNT_AllNulls"
```

Comment:
Could you put comma at the end of the line instead of at the start?

Reply:
Will move those commas. Good old habit (easier to comment out fields this way).